update icon documentation page with info about open source package (#16779)
microsoft/fluentui: apps/fabric-website/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

### Fluent UI React: Font-based icons

> The Fluent UI font-based icon set is released under the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license)

Reviewer: ```suggestion
The Fluent UI font-based icon set is released under the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license).
```

ReactDOM.render(<MyIconButton />, document.body.firstChild);
```

> ### Fluent UI React: SVG-based icons (Now Open Source in v8)

Reviewer: ```suggestion
### Fluent UI React: SVG-based icons (open source as of version 8)
```

Fluent UI recently released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.

> In v8, we are enabling more widespread use by open sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`. If you need branding icons, please refer to the package `@fluentui/react-icons-mdl2-branded`.

Reviewer: We shouldn't say "v8" on the website since it could be confused with the javascript engine v8. 
```suggestion

In `@fluentui/react` version 8, we are enabling more widespread use by open sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`. If you need branded icons, please refer to the package `@fluentui/react-icons-mdl2-branded`.
```

ReactDOM.render(<MyIconButton />, document.body.firstChild);
```

> ### Fluent UI React: SVG-based icons (open source as of version 8)

Reviewer: Since the whole text of the header gets used when generating the anchor link, it's probably best to either leave this text as-is (the suggestion) or update that anchor's URL referenced in FabricIconsPage.tsx.
```suggestion
### Fluent UI React: SVG-based icons
```

The Fluent UI SVG-based icon set is open source and released under the MIT license.

> Fluent UI released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.

Reviewer: ```suggestion
This icon set allows you to import and bundle only the icons you need, resulting in smaller download sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.
```

> In `@fluentui/react` version 8, we are enabling more widespread use by open sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`. If you need branded icons, please refer to the package `@fluentui/react-icons-mdl2-branded`.

Reviewer: If we're delayed in releasing the branded package we should temporarily remove the mention.
```suggestion
In `@fluentui/react` version 8, we are enabling more widespread use of icons by open-sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`.
```

### Fabric Core

> The Fabric Core icon set is released under the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license)

Reviewer: ```suggestion
The Fabric Core icon set is released under the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license).
```
